Cql/PackagerCLI/README.md

```diff
@@ -0,0 +1,62 @@
+# HL7 CQL Packager
+
+A .NET tool that packages CQL, ELM, and build outputs into a FHIR Library resource.
+
+## Overview
+
+The HL7 CQL Packager performs the following steps:
+
+1. **Translation**: Converts ELM JSON files into Linq.Expressions, then translates these expressions into C# code. Each ELM file is converted into a class, with each function within that file becoming a public method in the class.
+2. **Compilation**: Compiles the generated C# code into a single assembly per ELM file.
+3. **Packaging**: Packages the assembly (and optionally the original CQL, ELM, and C# source code) into a FHIR Library resource, creating one resource per original ELM file.
+
+## Getting Started
+
+### Installation
+
+Install the packager tool globally using the following command:
+
+```shell
+dotnet tool install Hl7.Cql.Packager --global --prerelease
```

Comment:
This command line program cannot be installed via the `dotnet tool install` yet, but it is a good idea to mention in a new ticket

Reply:
Why should this not be possible? I actually did this...

Reply:
I'm getting an unauthorized exception when running the dotnet tool install, probably something on my side. I created a ticket to follow up on this #572  and add to the documentation when this scenario occurs.

Reply:
I managed to update my source with an updated token. But now it says it can't find that tool on any of my source.

```cmd
~ dotnet tool install Hl7.Cql.Packager --global
hl7.cql.packager is not found in NuGet feeds https://api.nuget.org/v3/index.json, https://nuget.pkg.github.com/FirelyTeam/index.json, C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\.
```

Which sources do you have your side? `dotnet nuget list source`

Reply:
I eventually got it to work and documented my experience in #572 .